javascript/javascript1/week1/dog-age.js

```diff
@@ -0,0 +1,13 @@
+let dogYearOfBirth=2018;
+let dogYearFuture=2045;
+let dogYear=dogYearFuture-dogYearOfBirth;
+
+console.log(dogYear);
+let shouldShowResultInDogYears = true;
```

Comment:
Good job. 
just I would like you to use `const` not `let` here, 
because variables are never reassigned.